projects/core/components/dialog/dialogs.template.html

```diff
@@ -1,4 +1,4 @@
-<ng-container *ngIf="dialogs$ | async as dialogs">
+<ng-container *ngIf="dialogs() as dialogs">
```

Comment:
Let's drop the container and just use `dialogs()` multiple times.

Reply:
@waterplea https://github.com/taiga-family/taiga-ui/pull/8913#discussion_r1750107775